e2e-tests/micronaut-rest-api/src/main/resources/logback.xml

```diff
@@ -0,0 +1,16 @@
+<configuration>
```

Comment:
evomaster has been configured with log, might re-use it instead of a new configuration